src/components/Labelled/Labelled.tsx

```diff
@@ -36,6 +42,9 @@ export default function Labelled({
   helpText,
   children,
   labelHidden,
+  characterCount,
+  maxLength,
+  showCharacterCount,
```

Comment:
The `showCharacterCount` prop here feels unnecessary. As an implementor I would expect that if I pass a character count, it should show a character count. And if I don't pass a character count, it should not show a character count.